openslides/core/models.py

```diff
@@ -294,6 +297,59 @@ def __str__(self):
         return 'Message {}'.format(self.timestamp)
 
 
+class ProjectorMessage(RESTModelMixin, models.Model):
+    """
+    Model for messages.
+    """
+    access_permissions = MessageAccessPermissions()
+
+    message = models.TextField(blank=True)
+
+    identify = models.BooleanField(default=False)
+    """
+    If True, the message has priority and is displayed slightly different.
+    Also the message itself is irrelevant; the text is set to
+    "Projector {projectorid}: {projectorname}".
+    This is used for the identification of multiple projectors.
+    """
+
+    class Meta:
+        default_permissions = ()
+
+
+class ProjectorCountdown(RESTModelMixin, models.Model):
```

Comment:
The model can be named Countdown. I only wanted the model Message to be renamed to ProjectorMessage because we also have ChatMessage here. But there is only one Countdown. Sorry ...